charts/thoras/templates/crd/metricconnector.yaml

```diff
@@ -47,6 +47,12 @@ spec:
                       type: object
                     prometheus:
                       properties:
+                        query:
```

Comment:
Can we make the key name slightly more explicit about what it's for? What if we name it: `limit_to_single_metric`?